src/main/java/core/basesyntax/strategy/impl/PurchaseOperationHandler.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax.strategy.impl;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.enums.Operation;
+import core.basesyntax.strategy.mapvalidator.KeyPresentValidator;
+import java.util.Map;
+
+public class PurchaseOperationHandler
+        extends AbstractOperationHandler
+        implements KeyPresentValidator {
+    public PurchaseOperationHandler() {
+        super(Operation.PURCHASE);
+    }
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the redundant empty line after the class declaration.